### ISSUE TITLE
Convert the Verilog to Routing test runner from Perl to Python

### ISSUE THREAD

**Issue by mithro** [f4pga/ideas#16]
# Convert the test runner from Perl to Python

VtR has a [`vtr_flow`](https://github.com/verilog-to-routing/vtr-verilog-to-routing/tree/master/vtr_flow) test suite runner. Currently the [scripts are written in a mismash of Perl, Python and bash](https://github.com/verilog-to-routing/vtr-verilog-to-routing/tree/master/vtr_flow/scripts). It would be good if these scripts could all be rewritten in Python.

## Expected results

You can run the vtr test suite without requiring perl to be installed.

# Detailed Explanation

The major scripts which need to be converted are;
 - [ ] [run_vtr_task.pl](https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/vtr_flow/scripts/run_vtr_task.pl)
 - [ ] [run_vtr_flow.pl](https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/vtr_flow/scripts/run_vtr_flow.pl)

Some minor scripts which could be pretty easily converted are;
 - [ ] [blif2vhd.pl](https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/vtr_flow/scripts/blif2vhd.pl)
 - [ ] [vcd2act.pl](https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/vtr_flow/scripts/vcd2act.pl)
 - [More stuff can be found via this search](https://github.com/verilog-to-routing/vtr-verilog-to-routing/search?q=language%3APerl&unscoped_q=language%3APerl)

# Knowledge Prerequisites

 * Some Perl
 * Some Python

**Comment by ShraddhaDevaiya** [comment]
I would like to help in this if you permit?

**Comment by jgoeders** [comment]
We've started working on this

**Comment by sarahmaddox** [comment]
@jgoeders No worries! I'm not working on SymbiFlow at the moment, and don't have bandwidth to pick it up again. Please do go ahead with tackling this one.

**Comment by sarahmaddox** [comment]
@jgoeders Ah, I'm wondering if you meant to @-mention ShraddhaDevaiya rather than me?

**Comment by jgoeders** [comment]
Yes, sorry!  Wrong autocomplete.  @ShraddhaDevaiya.  Are you still planning to work on this?  If so, we should coordinate so we don't work on the same features.

**Comment by ShraddhaDevaiya** [comment]
Yeah If space is there , then definitely  want to work. and if possible can you please elaborate it more to get more idea about it.

**Comment by NP95** [comment]
@jgoeders  Is the rewrite done?

**Comment by jgoeders** [comment]
@NP95 There is now a run_vtr_flow.py in master.  https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1429

We are still working on converting the run_vtr_task.pl script.
